qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -131,7 +135,8 @@ def __init__(
             sample_from_iterations: If True, instead of taking the values in ``iterations`` as
                 powers of the Grover operator, a random integer sample between 0 and smaller value
                 than the iteration is used as a power, see [1], Section 4.
-            quantum_instance: A Quantum Instance or Backend to run the circuits.
+            quantum_instance: Pending deprecation\: A Quantum Instance or Backend to run the circuits.
```

Comment:
Was this `\` here on purpose? 🙂 
```suggestion
            quantum_instance: Pending deprecation: A Quantum Instance or Backend to run the circuits.
```